src/pages/gd/Swap/SwapCelo/index.tsx

```diff
@@ -1,65 +1,209 @@
-import { Web3Provider } from '@ethersproject/providers'
+import React, { useCallback } from 'react'
+
 import useActiveWeb3React from 'hooks/useActiveWeb3React'
-import React, { useEffect, useState } from 'react'
 import { useApplicationTheme } from 'state/application/hooks'
-import Swap from 'ubeswap-swap-dev'
+import { darkTheme, lightTheme, OnTxFail, OnTxSubmit, OnTxSuccess, SwapWidget, TokenInfo } from '@uniswap/widgets'
+import { Currency, CurrencyAmount } from '@uniswap/sdk-core'
 import { useConnectWallet } from '@web3-onboard/react'
-import { G$ContractAddresses, useGetEnvChainId } from '@gooddollar/web3sdk-v2'
+import { AsyncStorage, getDevice, G$ContractAddresses, useGetEnvChainId, useWeb3Context } from '@gooddollar/web3sdk-v2'
+import { useDispatch } from 'react-redux'
+import { addTransaction } from 'state/transactions/actions'
+import { ChainId } from '@sushiswap/sdk'
 
-interface AccountInfo {
-    account: string
-    explorerUrl: string
-    chainId: number
-    provider: Web3Provider
+const jsonRpcUrlMap = {
+    122: ['https://rpc.fuse.io', 'https://fuse-rpc.gateway.pokt.network'],
 }
 
-export const UbeSwap = (): JSX.Element => {
+const celoTokenList: TokenInfo[] = [
```

Comment:
whats this?

Reply:
The idea was to have more tokens added as option:
![image](https://github.com/GoodDollar/GoodProtocolUI/assets/6606028/74d0bb5b-5fb1-4c00-a55b-48c9b22dd9b4)
It just didn't get discussed yet which ones (if any) we wanted to add

Second thing is that by enabling the custom list (which we need for ourselves) for some reason the native celo shown doesn't have a logo.